desc/equilibrium.py

```diff
@@ -308,10 +308,10 @@ def from_near_axis(
                 elif spectral_indexing == "fringe":
                     L = 2 * M
             if N is None:
-                if N == np.inf:
-                    N = int((na_eq.nphi - 1) / 2)
-                else:
-                    N = M
+                N = M
+            if N == np.inf:
```

Comment:
Won't this never be true unless `M=np.inf`? I don't understand this change @f0uriest

Reply:
or if ``N is not None``

Reply:
You had it inside the if block, where we already have ``N is None`` so then it would never be `N == np.inf``

Reply:
Oh I see. I'm tired...